manifests/ta/input.pp

```diff
@@ -0,0 +1,13 @@
+define splunk::ta::input (
```

Comment:
If this is a private define, please comment as such in the header. And maybe use `assert_private()` from stdlib